src/pages/report/Report.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import i18n from "../../locales";
+import PageHeader from "../../components/page-header/PageHeader";
+import { useHistory } from "react-router";
+import { History } from "history";
+
+function goTo(history: History, url: string) {
+    history.push(url);
+}
+
+// function getConfig() {
```

Comment:
We usually remove commented code from the control version, to avoid having dead code.